tests/test_examples.py

```diff
@@ -7,8 +7,9 @@
 
 from ethpm_types import PackageManifest
 
-# NOTE: Must export GITHUB_ACCESS_TOKEN
-ETHPM_SPEC_REPO = github.Github(os.environ["GITHUB_ACCESS_TOKEN"]).get_repo("ethpm/ethpm-spec")
+ETHPM_SPEC_REPO = github.Github(os.environ.get("GITHUB_ACCESS_TOKEN", None)).get_repo(
+    "ethpm/ethpm-spec"
+)
```

Comment:
And just check this locally by running without the access token

Reply:
Verified that it does work without the `GITHUB_ACCESS_TOKEN` present in the environment